src/main/kotlin/io/snyk/plugin/settings/SnykProjectSettingsConfigurable.kt

```diff
@@ -98,8 +98,13 @@ class SnykProjectSettingsConfigurable(val project: Project) : SearchableConfigur
             snykProjectSettingsService?.additionalParameters = snykSettingsDialog.getAdditionalParameters()
         }
 
-        val params = DidChangeConfigurationParams(LanguageServerWrapper.getInstance().getSettings())
-        LanguageServerWrapper.getInstance().languageServer.workspaceService.didChangeConfiguration(params)
+        val wrapper = LanguageServerWrapper.getInstance()
```

Comment:
With my pending PR, we'll be able to completely move this to LanguageServerWrapper, as it has now a settings listener.